public/app/percona/backup/components/BackupInventory/BackupInventory.tsx

```diff
@@ -63,16 +66,16 @@ export const BackupInventory: FC = () => {
   );
   const styles = useStyles(getStyles);
 
-  const getData = async () => {
-    setPending(true);
+  const getData = async (showLoading = false) => {
```

Comment:
Yes. What happens is that we'll regularly update the table data (mainly because of status changes), but we don't want to be constantly putting a spinner and messing with what the user is seeing. This is achieved also using `autoResetExpanded` on the `Table`, which tells `react-table` to update the data but keep the table view as is (i.e. if a row is expanded, it will stay that way - with updated data if any -, which is not the default behaviour).